bin/yaml/swift.yaml

```diff
@@ -36,6 +36,17 @@ compilers:
           - '5.9'
           - '5.10'
           - '6.0.3'
+    dev-snapshot:
```

Comment:
This should be in the `if: nightly` section though, else it really will install every time anyone tries to install anything as opposed to our specific "nightly" install. In fact it looks like almost everything can be shared with the `nightly` section instead of copy pasting it?